### PR TITLE
Added graphs to metrics exemplar example and some nits.

### PR DESCRIPTION
**Pull request by Yun-Ting**
## Changes

Added pre-requisite, graphs, and update wordings of the readme file.

### CODE REVIEW

File: docs/metrics/exemplars/README.md [hunk 1]
```diff
@@ -50,15 +54,15 @@ directory and run the following command:
 dotnet run
 ```
 
-The above would start the application at port 5000. Navigate to
+The above command would start the application at port 5000. Navigate to
```
Comment (reyang) [inline docs/metrics/exemplars/README.md:57]: Maybe something like "Now the application is running, navigate to ... (the actual URL)"

File: docs/metrics/exemplars/README.md [hunk 2]
```diff
@@ -69,23 +73,27 @@ The application sends metrics (with exemplars), and traces to the OTel
 Collector, which export metrics and traces to Prometheus and Tempo
 respectively.
 
-Please wait 1-2 minutes before continuing so that enough data is generated and
-exported.
+Please wait for 1-2 minute(s) before continuing so that enough data is generated
```
Comment (reyang) [inline docs/metrics/exemplars/README.md:76]:
```suggestion
Please wait for 2 minutes before continuing so that enough data is generated
```

**Review by reyang — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Comment by cijothomas** [comment]
merging to keep making progress. we can iterate further, and do follow ups for the minor comments.

File: docs/metrics/exemplars/README.md (hunk 1 above)
Comment (Yun-Ting) [reply docs/metrics/exemplars/README.md:57]: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4195

**Pull request merged**